wikitemplate.md

```diff
@@ -145,6 +145,20 @@ _Each start should take less than 7 seconds_
   - [ ] Verify tips list is not lost after upgrade
   - [ ] Verify wallet panel transactions list is not lost after upgrade
 
+### Sync
```

Comment:
Saw you have upgrade tests for Android/iOS, do we want to test upgrade for laptop/desktop too?

Reply:
Ah nice catch. I'll add the same upgrade tests to desktop.

Reply:
sounds good. wasn't sure if it should be in Sync specific section like others or in the 'Update Tests' section. I guess either makes sense.